src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
+Animal.removeDeadAnimal = (animal) => {
+  Animal.alive = Animal.alive.filter(beast => beast !== animal);
+};
```

Comment:
Now you are comparing object to object(reference to reference)
I think @naziamoff was expecting to change the name of the method to `removeDeadAnimals` or `filterDeadAnimal`